src/nvmet.c

```diff
@@ -1054,15 +1135,24 @@ int nvmet_unexport_volume(char *device, char *host, int port, char *return_messa
 				}
 			}
```

Comment:
Do scan_ports() and then iterate through all /sys/kernel/config/nvmet/ports/<PORT>/subsystems/<NQN> and check output of access. If it exists, then we need to not continue with removing the block device from the subsystem. And then free linked list.

Reply:
I did this another way but it's done!